sdcm/utils/gce_utils.py

```diff
@@ -40,6 +40,9 @@
     'us-west1': 'abc'}
 
 
+SUPPORTED_PROJECTS = {'gcp', 'gcp-sct-project-1'} | {os.environ.get('SCT_GCE_PROJECT', 'gcp-sct-project-1')}
```

Comment:
just out of curiosity, the current project's name is `gcp`? not `skilled-adapter-452` ?

Reply:
it's the name of the configuration file we have on S3, since we have older branches that uses them. I don't want to rename it to match the actual project names